dandi/cli/cmd_delete.py

```diff
@@ -10,7 +10,11 @@
 @devel_debug_option()
 @map_to_click_exceptions
 def delete(paths, skip_missing, dandi_instance="dandi", devel_debug=False):
-    """ Delete Dandisets and assets from the server """
+    """Delete Dandisets and assets from the server
```

Comment:
There needs to be a period at the end of this line to terminate the short description shown in `dandi --help`.

Reply:
THANKS!